pkg/cli/topo.go

```diff
@@ -236,3 +237,78 @@ func readObjects(cmd *cobra.Command, args []string) ([]*topo.Object, error) {
 	}
 	return objects, nil
 }
+
+func getWatchEntityCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "entity <id> [args]",
+		Short: "Watch for topo changes",
+		RunE:  runWatchEntityCommand,
+	}
```

Comment:
you could put in max args 1, to handle <id>. Also I thin `<id>` should be `[id]` because its optional